BraintreeDropIn.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
                        Check out our development portal at https://developers.braintreepayments.com.
   DESC
   s.homepage         = "https://www.braintreepayments.com/how-braintree-works"
-  s.documentation_url = "https://developers.braintreepayments.com/ios/start/hello-client"
+  s.documentation_url = "https://developer.paypal.com/braintree/docs/start/hello-client/ios/v5"
```

Comment:
This CocoaPods Build was failing in CI due to this link being broken. Updating this link resolved the CI failure.

Reply:
Random:

Looks like if we exclude the version #, the link doesn't work (ex: https://developer.paypal.com/braintree/docs/start/hello-client/ios/). This used to work, but something probably broke w/ the docs migration.

(https://developer.paypal.com/braintree/docs/start/hello-client) <-- this works though 🤷

Reply:
Yeah that's odd - I'll ask docs about that